src/detect-engine-sigorder.c

```diff
@@ -860,8 +859,8 @@ static int SCSigOrderingTest01(void)
     DetectEngineCtxFree(de_ctx);
 
     return (i == 5);
- end:
-    return 0;
+
+    PASS;
```

Comment:
This statement will never execute because of the `return` statement that precedes it

Reply:
Do you think I should add a `FAIL_IF_NOT(i==5)` to replace this

Reply:
yes, that makes sense